packages/search/lib/indices/comments.js

```diff
@@ -19,18 +19,20 @@ module.exports = {
       'resolved.user.facebookId': {}
     },
     filter: {
-      bool: {
-        must: [
-          { term: { __type: type } },
-          { term: { published: true } },
-          { term: { adminUnpublished: false } }
-        ],
-        must_not: [
-          { terms: { discussionId: [
-            '3c625fe4-788f-44d5-ad5e-ac93bd9a6292' // Crowdfunding discussions
-          ] } }
-        ]
-      }
+      default: () => ({
+        bool: {
+          must: [
+            { term: { __type: type } },
+            { term: { published: true } },
+            { term: { adminUnpublished: false } }
+          ],
+          must_not: [
+            { terms: { discussionId: [
+              '3c625fe4-788f-44d5-ad5e-ac93bd9a6292' // Crowdfunding discussions
```

Comment:
off topic, but only realised now. This static ID is not so pretty to have in the code. How about `SEARCH_EXCLUDE_DISCUSSION_IDS` to configure dynamically?

Reply:
It is a good idea. Will follow.